emission/tests/netTests/TestAuthSelection.py

```diff
@@ -20,6 +20,7 @@
 import emission.net.auth.skip as enas
 import emission.net.auth.token_list as enat
 
+import bin.auth.insert_tokens as bai
```

Comment:
Moving the load/insert code into the wrapper will ensure that we don't need to import code from bin.